apps/spruce/cypress/integration/spawn/host.ts

```diff
@@ -251,23 +251,34 @@ describe("Navigating to Spawn Host page", () => {
 
     cy.getInputByLabel("Temporary Sleep Schedule Exemption").click();
     cy.get("td[aria-current=true]").as("currentDateCell");
+
     // Select a date in the future either this month or next month
     // if the current date is the last day of the month select the first day of the next month
     // if it is not select the next day
     // We can determine if the current date is the last day of the month if the next cell is disabled
-    cy.get("@currentDateCell")
-      .next()
-      .then(($el) => {
-        if (!$el || $el.attr("aria-disabled") === "false") {
-          cy.get("@currentDateCell").next().click();
-        } else {
-          cy.log("Current date is the last day of the month");
-          cy.get('button[aria-label="Next month"]').click();
-          cy.get('td[data-testid="lg-date_picker-calendar_cell"]')
-            .first()
-            .click();
-        }
-      });
+    cy.get("@currentDateCell").then(($currentDateCell) => {
```

Comment:
You might be able to simplify this logic and make the code path more deterministic by setting the browser date in cypress with [cy.clock](https://docs.cypress.io/api/commands/clock)

Reply:
I tried this out using `cy.clock()` and it had the unintended side effect of messing with how timers work on the page. Using it prevented the edit modal from even opening because it uses a timer to trigger opening the modal. I think this would introduce too much complexity to the page if we needed to handle managing all of the timers the page uses under the hood.

Reply:
Thanks for checking it out!

Reply:
Great suggestion tho! I didn't know about this API. I think it's more geared towards component testing